src/navigate-pop/launching-applications.md

```diff
@@ -0,0 +1,92 @@
+<!--Sources
+Pop Training Guide (Google doc)
+https://support.system76.com/articles/pop-basics/
+
+-->
+# Locate, Launch, and Navigate Between Applications
+
+---
+
+## Browsing Installed Applications
+
+Installed applications can be viewed in the Applications menu and the Pop!\_Shop Installed screen.
+
+### The Applications Menu
+
+The Applications menu can be accessed from the Dock or by clicking `Applications` in the upper left corner of the screen.
+
+![Launch Applications Menu](/images/locate-launch-applications/launch-applications-menus.png)
+
+Applications can be grouped into folders. Click `Create Folder` and give the folder a name.
+
+![Create Application Folder](/images/locate-launch-applications/create-application-folder.png)
+
+Click and drag an application icon to add it to your folder.
+
+<video autoplay loop>
+    <source src="/images/locate-launch-applications/drag-application.webm" />
+</video>
+
+### The Pop!\_Shop
+
+Launch the Pop!\_Shop, then click the `Installed` button.
+
+![Pop Shop Installed Apps](/images/locate-launch-applications/pop-shop-installed-apps.png)
+
+## Launching Applications
+
+Applications can be launched from the Dock, using the Launcher, the Applications menu, and from the Terminal.
+
+### Using the Launcher
+
+Press the `SUPER` key or the Show Launcher icon to bring up the Launcher.
+
+![Show Launcher](/images/locate-launch-applications/show-launcher.png)
+
+Search for the name of the application, then hit `Enter`.
+
+![Search App Name](/images/locate-launch-applications/search-app.png)
+
+### Using the Applications Menu
+
+Click the Applications menu icon located in the Dock.
+
+![Click Applications Dock](/images/locate-launch-applications/click-applications-dock.png)
+
+Click the application's icon to launch it.
+
+![Click App Icon](/images/locate-launch-applications/click-app-icon.png)
```

Comment:
What's that circle on the top left of the Settings icon?

Reply:
I have yet to find a screenshotting tool that does not remove focus when activated. I use a timed screenshot and crop in Gimp, what you're seeing is my Gimp selection tool. I'll retake this one.